lms/djangoapps/grades/api/views.py

```diff
@@ -3,6 +3,10 @@
 
 from django.contrib.auth import get_user_model
 from django.http import Http404
+from edx_rest_framework_extensions.permissions import (
```

Comment:
These changes need to be included in the v1 of the Grades API - and not part of v0.

Reply:
Nice, Catch!